crypto/sigpb/sigpb.proto

```diff
@@ -44,10 +44,24 @@ message DigitallySigned {
     ECDSA = 3;
   }
 
+  // SignatureCipherSuite defines the set of algorithms used for signing.
+  enum SignatureCipherSuite {
```

Comment:
Have you considered moving this message to the top level? It's used in other contexts, outside of DigitallySigned, so it could stand on its own.

(Same would apply for HashAlgo and SigAlgo, assuming we wouldn't delete them.)